StortSpelProjekt/Engine/src/Renderer/Renderer.cpp

```diff
@@ -2104,8 +2111,8 @@ void Renderer::removeComponents(Entity* entity)
 
 void Renderer::prepareScenes(std::vector<Scene*>* scenes)
 {
-	prepareCBPerFrame();
-	prepareCBPerScene();
+	SubmitUploadPerFrameData();
```

Comment:
Should be more comments to explain stuff like this. Why is the function called twice?

Reply:
Nevermind, is not the same function :^)